content/docs/reference/common_attributes.md

```diff
@@ -0,0 +1,200 @@
+
+Example: 
+
+[sourcecode:html]
+<amp-img src="/img/amp.jpg"
+    width="1080"
+    height="610"
+    layout="responsive"
+    alt="an image">
+</amp-img>
+[/sourcecode]
+
+For more information, see [Layout & Media queries](https://www.ampproject.org/docs/guides/responsive/control_layout) and the [Layout Spec](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-layout.md).
+
+## media
+
+All AMP elements support the `media` attribute. The value of `media` is a media query. If the query does not match, the element is not rendered and its resources and potentially its child resources will not be fetched. If the browser window changes size or orientation, the media queries are re-evaluated and elements are hidden and shown based on the new results.
+
+Example:
+
+[sourcecode:html]
+<amp-img
+    media="(min-width: 650px)"
+    src="wide.jpg"
+    width="466"
+    height="355" layout="responsive"></amp-img>
+<amp-img
+    media="(max-width: 649px)"
+    src="narrow.jpg"
+    width="527"
+    height="193" layout="responsive"></amp-img>
+[/sourcecode]
+
+For more information, see [Layout & Media queries](https://www.ampproject.org/docs/guides/responsive/control_layout).
+
+## noloading
+
+The `noloading` attribute indicates whether the "loading indicator" should be turned **off** for this element. Many AMP elements are white-listed to show a "loading indicator", which is a basic animation that shows that the element has not yet fully loaded. 
+
+Often used with: images, animations, videos, and ads
+
+Example: 
+
+[sourcecode:html]
+<amp-img src="card.jpg"
+    noloading
+    height="190"
+    width="297"
+    layout="responsive">
+</amp-img>
+[/sourcecode]
+
+## on
+
+The `on` attribute is used to install event handlers on elements. The events that are supported depend on the element.
+
+Often used with: lightboxes, sidebars, live lists, and forms
+
+Syntax:
+
+[sourcecode:text]
+eventName:targetId[.methodName[(arg1=value, arg2=value)]]
+[/sourcecode]
+
+Example:
+
+[sourcecode:html]
+<button on="tap:my-lightbox">Open lightbox</button>
+<amp-lightbox id="my-lightbox" layout="nodisplay">
+  ...
+</amp-lightbox>
+[/sourcecode]
+
+For more information, see  [Actions and Events in AMP](https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-events.md).
+
+## placeholder
+
+The `placeholder` attribute can be set on any HTML element, not just AMP elements. The placeholder attribute indicates that the element marked with this attribute acts as a placeholder for the parent AMP element. If specified, a placeholder element must be a direct child of the AMP element.
```

Comment:
The first sentence here and for `fallback` is a bit misleading.

Maybe lead with what they are for and then say "The attribute can be placed on any HTML element that is a direct child of an amp element that supports placeholders."